src/jsparse.c

```diff
@@ -2303,7 +2303,8 @@ NO_INLINE void jspeBlockNoBrackets() {
   JsVar *oldBlockScope = jspeBlockStart();
   if (JSP_SHOULD_EXECUTE) {
     while (lex->tk && lex->tk!='}') {
-      JsVar *a = jspeStatement();
+      bool goAgain;
```

Comment:
should there be a default value?

Reply:
I've been using it as an out (only) param, but appreciate that might not be the style